package.json

```diff
@@ -3,38 +3,25 @@
   "version": "1.0.0",
   "description": "Leather wallet shared code",
   "scripts": {
-    "install:clean": "rm -rf \"**/node_modules\" && pnpm -r clean",
-    "install:fresh": "pnpm install:clean && pnpm i",
-    "install:nuke": "rm -rf pnpm-lock.yaml && pnpm install:fresh",
-    "lint:eslint": "eslint .",
-    "lint:eslint:fix": "pnpm lint:eslint --fix",
-    "lint:prettier": "prettier --check \"*.{ts,tsx,js,json}\"",
-    "lint:prettier:fix": "prettier --write \"*.{ts,tsx,js,json}\" *.js",
-    "lint:types": "tsc --noEmit",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "lint-projects:all": "nx run-many -t format,lint,typecheck --parallel=1",
+    "lint-projects:affected": "npx nx affected -t format,lint,typecheck --parallel=1 --base=dev",
+    "commitlint": "commitlint"
   },
   "keywords": [],
   "author": "",
   "license": "MIT",
+  "workspaces": [
+    "packages/*"
+  ],
   "devDependencies": {
-    "@changesets/cli": "^2.26.2",
     "@commitlint/cli": "^18.0.0",
     "@commitlint/config-conventional": "^18.0.0",
-    "@leather-wallet/eslint-config": "workspace:^",
-    "@leather-wallet/prettier-config": "workspace:^",
-    "@leather-wallet/tsconfig-config": "workspace:^",
-    "eslint": "^8.52.0",
     "husky": "^8.0.3",
-    "prettier": "^3.0.3",
-    "syncpack": "^11.2.1",
-    "typescript": "^5.2.2",
-    "lint-staged": "^15.0.2"
+    "nx": "^17.1.1"
   },
   "engines": {
     "node": ">=20.3.0",
-    "pnpm": ">=7.10.0",
-    "npm": "please-use-pnpm",
-    "yarn": "please-use-pnpm"
-  },
-  "packageManager": "pnpm@8.7.5"
+    "npm": ">=10"
```

Comment:
Maybe we could keep the enforcement here but for `npm` instead of `pnpm`? Not a big deal really

Reply:
Done!